README.rst

```diff
@@ -1 +1,5 @@
 This is the Artefactual Storage Service for Archivematica.
+
+Start worker:
+
+celery -A storage_service worker -l info
```

Comment:
This is being done in this branch of the ansible role (not merged to master yet) https://github.com/artefactual-labs/ansible-archivematica-src/tree/dev/ss-celery-worker